.travis.yml

```diff
@@ -2,14 +2,15 @@ language: scala
 jdk:
   - oraclejdk8
 scala:
-  - 2.11.7
-sudo: false
-
-install: ./gradlew assemble -x signArchives
+  - 2.12.3
 
-script:
-  - ./gradlew check -x signArchives
+sudo: false
 
+before_cache:
+  - find $HOME/.ivy2/cache -name "ivydata-*.properties" -print -delete
```

Comment:
I just followed the sbt docs for travis: http://www.scala-sbt.org/1.0/docs/Travis-CI-with-sbt.html